fe/fe-core/src/main/java/org/apache/doris/catalog/Env.java

```diff
@@ -1679,10 +1679,15 @@ private void transferToMaster() {
      */
     void advanceNextId() {
         long currentId = idGenerator.getBatchEndId();
-        long currentNanos = System.nanoTime();
+        long currentMill = System.currentTimeMillis();
         long nextId = currentId + 1;
-        if (nextId < currentNanos) {
-            nextId = currentNanos;
+        // Reserve at most 8 billion for use in case of bugs or frequent reboots (21 billion reboots)
+        if ((1L << 63) - nextId < (1L << 33)) {
```

Comment:
```suggestion
        // Reserve ~1 trillion for use in case of bugs or frequent reboots (~2 billion reboots)
        if ((1L << 63) - nextId < (1L << 40)) {
```